localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -199,6 +203,7 @@ class NDTScanMatcher : public rclcpp::Node
 
   std::mutex ndt_ptr_mtx_;
   std::unique_ptr<SmartPoseBuffer> initial_pose_buffer_;
+  std::optional<geometry_msgs::msg::Point> current_position_ = std::nullopt;
```

Comment:
In the comments, please describe that it is only used when `use_dynamic_map_loading_` is enabled.
Or rename it to something more specific.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5915/commits/dae9c048805681ea832be4fcc5ba4db495841dad

I also noticed that this variable was causing a data race, so I added std::mutex https://github.com/autowarefoundation/autoware.universe/pull/5915/commits/4313c2e2c3855ef837d1ff4c43692aef0db4083b